src/models/scatter.js

```diff
@@ -535,9 +566,15 @@ nv.models.scatter = function() {
             })
             .watchTransition(renderWatch, 'scatter points')
             .attr('transform', function (d) {
-                return 'translate(' +
-                    nv.utils.NaNtoZero(x(getX(d[0], d[1]))) + ',' +
-                    nv.utils.NaNtoZero(y(getY(d[0], d[1]))) + ')'
+                // Mirror Chart
+                var yValueTemp = y0(getY(d[0],d[1]));
+                if (d[0].position === 'above') {
+                    yValueTemp = y0Above(getY(d[0],d[1]));
+                }
+                if (d[0].position === 'below') {
+                    yValueTemp = y0Below(getY(d[0],d[1]));
+                }
+                return 'translate(' + nv.utils.NaNtoZero(x0(getX(d[0],d[1]))) + ',' + nv.utils.NaNtoZero(yValueTemp) + ')';
```

Comment:
For the record...

A line chart in nvd3 is built using two charting sub-models, the `line` and `scatter` models. When we plot time series data using a line chart we are actually drawing an svg line and a series of point for every datum on the series. By default nvd3 shows the line but hides the points. Here's a screen cap where I have tweaked the CSS to show the scatter points hidden by nvd3.

<img src="https://user-images.githubusercontent.com/17259166/29644110-7be6ea28-8828-11e7-87c2-3e5ef93bc399.png" width="500">

When the user rolls-over a point nvd3 will then show the closest point to the mouse pointer:

<img src="https://user-images.githubusercontent.com/17259166/29644151-bb67bc40-8828-11e7-9eee-266d04967256.png" width="500">

The code highlighted above is the only addition I had to make in order to get the mirrored mode of the line chart working properly, otherwise the points would be positioned using the total height of the chart as shown above:

<img src="https://user-images.githubusercontent.com/17259166/29644254-8b2f12ca-8829-11e7-9921-8b15f9e78686.png" width="500">